tests/zfs-tests/tests/functional/quota/quota.kshlib

```diff
@@ -62,7 +62,7 @@ function fill_quota
 
 	typeset -i file_size=`$LS -ls $MNTPT/$TESTFILE1 | $AWK '{ print $1 }'`
 	typeset -i limit=0
-	(( file_size = file_size * 512 ))
+	(( file_size = file_size * $BLOCK_SIZE ))
```

Comment:
Thank you for your guidance! In the first time i have thought to modify in this way, but when debugging this test script find that  ls -ls command print the testfile size in blocks based its write blocksize(8192) rather than Linux default blocksize(1024), the write file and ls -ls print result as following:

write file:
14:00:04.93 /var/tmp/testdir19626/file1: `block_size = 8192`, write_count = 20000000, offset = 0, data = 0->120 
14:00:04.93 write failed (-1), `good_writes = 1217`, error: Disk quota exceeded[122]

ls -ls:
14:00:04.94 NOTE: `1233` -rw-r--r-- 1 root root `1217` Oct 12 14:00 /var/tmp/testdir19626/file1

from above log we can lean that ls -ls used the same blocksize(8192) as when testfile writing, and I still can't confirm what causes this performance.  Do you have any good Suggestions! thanks.